amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -2964,6 +2964,19 @@ type ColonyAction @model @searchable {
   Applicable to `EDIT_EXPENDITURE` action only
   """
   expenditureSlotChanges: ExpenditureSlotChanges
+  """
+  Approved tokens impacted by the action (used for manage tokens)
+  """
+  approvedTokenChanges: ApprovedTokenChanges
+}
+
+"""
+Token IDs
+"""
+type ApprovedTokenChanges {
+  added: [ID!]
```

Comment:
if you want to avoid this being potentially null, we can do `added: [ID!]!` and we won't need to check if the array is defined.
Not sure if it's applicable design wise, feel free to just resolve this in that case

Reply:
Good catch! I have updated this and removed the now unnecessary checks.